### PR TITLE
Fixes #121 Added Ask.com support

### LINKED ISSUE #121: Adding ask.com support
This will be an enhancement to the existing project

### PR DESCRIPTION
**Pull request by sayak119**
Fixes #121 , #165 

- [x] Added Ask.com support 
- [x] Added Ask.com icon
- [x] Python 3 compatibility
- [x] Taken care of all the spaces and other specifications

### CODE REVIEW

**Comment by sayak119** [comment]
@Pk13055 @vaibhavsingh97 @nikhilrayaprolu Please review. Sorry for inconvenience. My local repo got messed up. Sorry again.

File: app/scraper.py [hunk 1]
```diff
@@ -158,6 +164,31 @@ def yahoo_search(query,count):
...
+            [[Tile1,url1], [Title2, url2],..]
+    """
+    urls=[]
+    response = get_ask_page(query)
+    soup = BeautifulSoup(response.text, 'html.parser')
+    for div in soup.findAll('div', {'class': 'PartialSearchResults-item'}):
+        title = div.div.a.text
+        url = div.div.a['href']
+        urls.append({'title': title, 'link': url})
+
```
Comment (vaibhavsingh97) [inline app/scraper.py]: Remove spaces

File: app/scrapers/ask.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                [[Tile1,url1], [Title2, url2],..]
+        """
+        urls=[]
+        response = get_ask_page(query)
+        soup = BeautifulSoup(response.text, 'html.parser')
+        for div in soup.findAll('div', {'class': 'PartialSearchResults-item'}):
+            title = div.div.a.text
+            url = div.div.a['href']
+            urls.append({'title': title, 'link': url})
+
```
Comment (vaibhavsingh97) [inline app/scrapers/ask.py]: Remove spaces

Comment (sayak119) [reply app/scrapers/ask.py]: where should i remove spaces

File: app/scraper.py (hunk 1 above)
Comment (vaibhavsingh97) [reply app/scraper.py]: Line 190

File: app/scrapers/ask.py (hunk 2 above)
Comment (vaibhavsingh97) [reply app/scrapers/ask.py]: Line 36

**Comment by vaibhavsingh97** [comment]
@sayak119 Why the commits are mixed up?

**Pull request closed without merging**